xml/System/UIntPtr.xml

```diff
@@ -767,8 +767,17 @@
       </ReturnValue>
       <Docs>
         <summary>Represents the largest possible value of <see cref="T:System.UIntPtr" />.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <value>
+          <paramref name="x" /> if it is greater than <paramref name="y" />; otherwise, <paramref name="y" />.</value>
```

Comment:
This value doesn't seem to be correct. There are no parameters.

Reply:
This looks to have been a mis-resolved merge conflict in the source.